doc/md/cli/projects.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Project Structure
+id: projects
+slug: /cli/projects
+---
+### Atlas Project Files
+
+Project files provide a convenient way to describe and work with different
+environments which are managed using Atlas.  A project file is a file named
+`atlas.hcl` which is located at the root of the project and contains one or 
+`env` blocks. For example:
+
+```hcl
+// Define an environment named "local"
+env "local" {
+  // Declare where the schema definition file resides.
+  src = "./schema/project.hcl"
+  
+  // Define the URL of the database which is managed in
+  // this environment.
+  url = "mysql://localhost:3306"
+  
+  // Define the URL of the Dev Database for this environment
+  // See: https://atlasgo.io/dev-database
+  dev = "mysql://localhost:3307"
+  
+  // The schemas in the database that are managed by Atlas.
+  schemas = ["users", "admin"]
+}
+
+env "dev" {
+  // ... a different env
+}
+```
+
+Once defined, a project's environment can be worked against using the `--env` flag. 
+For example:
+
+```shell
+$ atlas schema apply --env local
```

Comment:
```suggestion
atlas schema apply --env local
```
We have a copy button.

Reply:
I see now that it exists in other places. I'll send a patch to fix it in all other places.